services/search/api.py

```diff
@@ -114,29 +118,21 @@ def to_representation(self, obj):
             names["en"] = getattr(obj, "name_en")
             representation["name"] = names
 
-        if self.context["extended_serializer"]:
-            if object_type == "unit":
-                representation["street_address"] = getattr(obj, "street_address")
-                if hasattr(obj.municipality, "id"):
-                    representation["municipality"] = getattr(obj.municipality, "id")
-                try:
-                    shortcomings = obj.accessibility_shortcomings
-                except UnitAccessibilityShortcomings.DoesNotExist:
-                    shortcomings = UnitAccessibilityShortcomings()
-                representation[
-                    "accessibility_shortcoming_count"
-                ] = shortcomings.accessibility_shortcoming_count
-                representation["contract_type"] = UnitSerializer.get_contract_type(
-                    self, obj
-                )
-                representation["department"] = DepartmentSerializer(obj.department).data
-                if obj.geometry:
-                    representation["geometry"] = munigeo_api.geom_to_json(
-                        obj.geometry, DEFAULT_SRS
-                    )
-                else:
-                    representation["geometry"] = None
-
+        if object_type == "unit":
```

Comment:
@juuso-j Check indentation here. I think `unit` and `address` should be in the same level here.